docs/source/api/metasyn.distribution.rst

```diff
@@ -1,13 +1,8 @@
 metasyn.distribution package
-==============================
+============================
 
-.. automodule:: metasyn.distribution
-   :members:
-   :undoc-members:
-   :show-inheritance:
-
-Submodules
-----------
+The :mod:`metasyn.distribution` package consists of several distribution modules. 
```

Comment:
Isn't this supposed to go in the docstrings?

Reply:
Nope, because of the way the API is structured now it wouldn't show up.

Reply:
Can you explain why we can't have the docstring in the metasyn.distribution.__init__ can't be shown in its place there?

Reply:
No okay, technically it is possible. But I have restructured the API page in a way that the modules are listed individually, instead of being listed by autodocumenting the distribution package, as such it wouldn't show. 

I did this because the page used to list the modules in both ways (through the distribution package and individually), and was very long, redundant and mostly confusing as a result. And because it gives flexibility, as it now allows me to change the order of the modules (putting 'base' first), and add extra text or formatting as need be. It also allows for neater sidebar formatting.

That said I realise this is not as easy to maintain going forward, so I will change it to autodoc the whole distribution package, and I'll move the text to the module docstring while I'm at it.

Reply:
Never mind, I have looked at it - and the overall flexibility and better organizing of the page is just a lot better when listing all the modules individually. I think the slight increase in maintainability effort is worth it (it also significantly impacts sidebar navigation), so it will have to stay as a separate string at the top of the doc.

Reply:
Okay, if it works out better, I'm okay with it.